sample-compose/app/build.gradle.kts

```diff
@@ -177,3 +179,46 @@ dependencies {
     androidTestImplementation("androidx.test:rules:${Versions.TEST_RULES_VERSION}")
     androidTestImplementation("io.mockk:mockk-android:${Versions.TEST_MOCKK_VERSION}")
 }
+
+/*
+ * Kover configs
+ */
+
+dependencies {
+    kover(project(":data"))
+    kover(project(":domain"))
+}
+
+koverReport {
+    defaults {
+        mergeWith("stagingDebug")
+        mergeWith("stagingRelease")
```

Comment:
I think, by default, the project should only support `debug` flavor, as we do not often have custom code in `release` flavor. Merging with `release` flavor will duplicate this operation.

![image](https://github.com/nimblehq/android-templates/assets/16315358/526120d8-a48a-45d6-ad11-5cd8b1e6b7a6)


```suggestion
        mergeWith("stagingDebug")
```